content/for-developers/strategies-and-conventions.mdx

```diff
@@ -66,11 +66,11 @@ lastUpdatedDate: 2022-04-06T12:00:00.000Z
     </Section>
 
     <Section heading="Centered container and screen edge" id="centered-container-and-screen-edge">
-      In Decanter, we use a fluid “[centered container](https://github.com/SWS/decanter/blob/v7/src/plugins/components/layout/centered-container.js)” (`cc` class) to constrain the width of the content area on a website by setting proportional “screen edge” at each breakpoint.
+      In Decanter, we use a fluid “[centered container](https://github.com/SU-SWS/decanter/blob/main/src/plugins/components/layout/centered-container.js)” (`cc` class) to constrain the width of the content area on a website by setting proportional “screen edge” at each breakpoint.
 
       Screen edge is the minimal amount of space between the outside edge of this centered container and the edge of the browser window. Its margins are set at each breakpoint, up to and not including the largest breakpoint (2XL). The total width of the centered container is not fixed, but rather expands (fluid) as the viewport expands up to 1700px. At which point, the centered container reaches its maximum width of 1500px with screen edge of 100px margin on either side of the container.
 
-      [Screen edge values](https://github.com/SWS/decanter/blob/9442bddc27bd699acdb25c25a1b84b07f905245a/src/plugins/theme/decanter.js#L47) are curated and do not follow a mathematical pattern:
+      [Screen edge values](https://github.com/SU-SWS/decanter/blob/9442bddc27bd699acdb25c25a1b84b07f905245a/src/plugins/theme/decanter.js#L47) are curated and do not follow a mathematical pattern:
```

Comment:
@yvonnetangsu do you want this to point to a branch or a commit?

Reply:
> @yvonnetangsu do you want this to point to a branch or a commit?

@jbickar Hmm master branch is probably ok now that v7 is default - thanks!

Reply:
Thanks for interpreting `master` = `main` branch @jbickar. My brain is still in Japan time zone 😆